alexa.utterances

```diff
@@ -525,6 +514,41 @@ PlayerZoomOutMoveUp zoom out and pan up
 PlayerZoomOutMoveUp zoom out and track up
 PlayerZoomReset normal zoom
 PlayerZoomReset reset zoom
+PlayLatestEpisode play latest episode of {Show}
+PlayLatestEpisode play newest episode of {Show}
+PlayLatestEpisode play the latest episode of {Show}
+PlayLatestEpisode play the newest episode of {Show}
+PlayLatestEpisode watch latest episode of {Show}
+PlayLatestEpisode watch newest episode of {Show}
+PlayLatestEpisode watch the latest episode of {Show}
+PlayLatestEpisode watch the newest episode of {Show}
+PlayMovie play film {Movie}
+PlayMovie play movie {Movie}
+PlayMovie watch film {Movie}
+PlayMovie watch movie {Movie}
+PlayNextEpisode play next episode of {Show}
+PlayNextEpisode play the next episode of {Show}
+PlayNextEpisode watch next episode of {Show}
+PlayNextEpisode watch the next episode of {Show}
+PlayPause pause
+PlayPause play
+PlayPause resume
+PlayRandomEpisode play a episode of {Show}
+PlayRandomEpisode play a random episode of {Show}
+PlayRandomEpisode play episode of {Show}
+PlayRandomEpisode play random episode of {Show}
+PlayRandomEpisode watch a episode of {Show}
+PlayRandomEpisode watch a random episode of {Show}
+PlayRandomEpisode watch episode of {Show}
+PlayRandomEpisode watch random episode of {Show}
+PlayRandomMovie play a random film
+PlayRandomMovie play a random movie
+PlayRandomMovie play random film
+PlayRandomMovie play random movie
+PlayRandomMovie watch a random film
+PlayRandomMovie watch a random movie
+PlayRandomMovie watch random film
+PlayRandomMovie watch random movie
 Prev listen to previous
```

Comment:
Why did you move these?  This file was recently auto-generated, and the next time it is, they'll just move back anyway.